gsw/geostrophy.py

```diff
@@ -295,7 +295,7 @@ def geostrophic_velocity(geo_strf, lon, lat, p=0, axis=0):
 
     if lon.shape != lat.shape or lon.ndim != 1:
         raise ValueError('lon, lat must be 1-D and matching; found shapes'
-                         ' %s and %s' % (lon.shape, lat.shape))
+                         ' {} and {}'.format(lon.shape, lat.shape))
 
     if geo_strf.ndim not in (1, 2):
         raise ValueError('geo_strf must be 1-D or 2-d; found shape %s'
```

Comment:
I wonder why the old percent syntax is changed above, but not here?
There's actually nothing wrong with the old  syntax; it isn't going to go
away.  But in most cases, f-strings are more readable.  I would certainly
use them for new code.

Reply:
This comment somehow got connected to the wrong line.  A few lines below here there is an unchanged line that has the percent syntax, and presumably ruff missed it.

Reply:
Probably a bug in pyupgrade via ruff. I'll investigate and report upstream.

Reply:
Looks like we are in the situation in the note here: https://github.com/asottile/pyupgrade#f-strings